Tools/Scripts/webkitpy/benchmark_runner/run_benchmark.py

```diff
@@ -55,6 +55,7 @@ def config_argument_parser():
     parser.add_argument('--diagnose-directory', dest='diagnose_dir', default=diagnose_directory, help='Directory for storing diagnose information on test failure. Defaults to {}.'.format(diagnose_directory))
     parser.add_argument('--no-adjust-unit', dest='scale_unit', action='store_false', help="Don't convert to scientific notation.")
     parser.add_argument('--show-iteration-values', dest='show_iteration_values', action='store_true', help="Show the measured value for each iteration in addition to averages.")
+    parser.add_argument('--profile-output-directory', dest='profile_output_directory', default=None, help="Collect LLVM profiles for PGO, and copy them to this directory.")
```

Comment:
Could this be part of `diagnose-directory` or we don't always want this to run?

Reply:
This should only run rarely

Reply:
In that case, would it make more sense instead to have a `--collect-pgo-profiles` flag that turns on the collection and stores them in `diagnose_dir`? Or is it important that these are collected to a separate directory?